Sources/Common/DataModel/Triangle/index.js

```diff
@@ -69,6 +69,7 @@ function vtkTriangle(publicAPI, model) {
     if (n[0] !== 0 || n[1] !== 0 || n[2] !== 0) {
       // Intersect plane of triangle with line
       const plane = vtkPlane.intersectWithLine(p1, p2, pt1, n);
```

Comment:
it is too bad that `vtkPlane` returns a key named `intersection` and not `intersect`.
That would be more consistent with `vtkLine`, `vtkTriangle`, `vtkCellPicker`.

@jourdain WDYT is it worth it to rename `intersection` into `intersect` (or inversely)? That would be a breaking change though.